src/app/pages/challenge/challenge-overview/challenge-overview.component.ts

```diff
@@ -51,27 +59,26 @@ export class ChallengeOverviewComponent implements OnInit {
       )
       .subscribe((organizerList: ChallengeOrganizerList) => {
         const organizers = organizerList.challengeOrganizers;
-        this.organizers = organizers;
-        this.dataSource =
+        this.organizers =
           organizers.length > 0
             ? new MatTableDataSource(organizers)
             : undefined;
         setTimeout(() => {
-          if (this.dataSource) {
-            this.dataSource.sort = this.sort;
+          if (this.organizers) {
+            this.organizers.sort = this.sort;
           }
         });
       });
 
     this.readme$ = this.challengeDataService.getReadme();
   }
 
-  // TODO: use avatarUrl once we can access
-  getAvatar(name: string): Avatar {
+  // TODO: use avatarUrl of sponsor once we can access
+  getAvatar(name: string, size: number): Avatar {
     return {
       name: name,
       src: '',
-      size: 30,
+      size: size,
     };
   }
```

Comment:
@tschaffter I thought sponsors/organizers will have avatarUrl eventually. 

I will update this part soon to look up the account object to get avatar.